### DISCUSSION TITLE
Performance adjustments

### DISCUSSION THREAD

**Discussion by EpicChrisplay** [official-antistasi-community/A3-Antistasi#2783] [Q&A]
Hey guys,

I setup my own Antistasi server and it is running quite good so far.

I just noticed that the performance on the client side seems to drop quite much (only getting 20-30 fps in the cities or at outposts).
While the server seem to run very smooth an 90-110 fps on his side, it seems that the clients can't handle the mission as well.

I already tried to reduce "Civ Count" and "Spawn Distance", with very little effect.
Also tried the mod "Arma 3 Performance Extension".

Is there something that can be done to balance the workload between the clients and the server?
Are there server or client sided settings that can help with that?

I really appreciate any help with this <3
Cheers
Chris

**Comment by jaj22** [comment]
It's just Arma. Client perf depends largely on how how many units (including dead ones) and vehicles there are. Visible is worse than invisible, but invisible isn't free either. Best you can do is reduce draw distance (Arma), reduce spawn distance (Antistasi) and don't spam rebel AIs more than necessary.

20-30fps is bad though. My old 4790 manages that on 30-player community servers, with twice the frame rate on low player count servers. Maybe you have some performance-murdering mods that you've overlooked.

**Reply by EpicChrisplay** [reply]
The installed mods aren't that exotic: Antistasi, CBA_A3, ace, JSRS and the performance mod I mentioned.

Thing is I lowered the antistasi settings as far a possible and currently test on completely fresh game sessions with only me online. So no dead bodies, spawned vehicles or rebel AI.

Also my computer isn't at its limit at all. Not even with one CPU Thread. (i7 7700K, 32 GB Ram, M.2 SSD)
Already tweaked the client with startup parameters and tried to lower my ingame video settings.
The server doesn't get that much workload, too.

I would really like to play it with my friends. But I worry that this won't do for them.

**Reply by jaj22** [reply]
If it's a fresh game session then that's clearly not normal. Antistasi runs next to nothing on the client until you recruit AIs. Even localhost on my hardware I'll get way above 60fps on a clean spawn. It's basically the same as dropping yourself into the same location in the editor.

**Reply by EpicChrisplay** [reply]
I get 50-70 fps at the HQ. When I drive/walk near enemy outposts/cities it starts to drop to around 30, sometimes even further for a moment.

**Comment by rautamiekka** [comment]
Is this with a Dedicated Server or through the multiplayer menu ?

**Reply by EpicChrisplay** [reply]
It's a dedicated server on another computer of mine.
Runs with a i7-950 I think and 12GB RAM on a cheap SSD.
